crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -176,3 +221,444 @@ impl From<infra::persistence::DatabaseError> for Error {
         Self::Infra(err.0)
     }
 }
+
+#[cfg(test)]
```

Comment:
Two old tests just moved: `settlement` and `settlement_with_protocol_fee`
One new test: `settlement_with_cow_amm`